pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -111,14 +117,40 @@ func GetTemplateData(config *baremetal.Platform, networks []types.MachineNetwork
 	templateData.ExternalStaticDNS = config.BootstrapExternalStaticDNS
 	templateData.ExternalMACAddress = config.ExternalMACAddress
 
-	_, externalURLv6 := externalURLs(config.APIVIPs)
+	// If the user has manually set disableVirtualMediaTLS to False in the Provisioning CR,
+	// then enable TLS in ironic
+	templateData.DisableIronicVirtualMediaTLS = true
```

Comment:
Let's also note that the default of `true` is a temporary condition for the sake of backporting the change and that we'll change it to `false` eventually.